config/default/common/config/wv.json/conceptIds.json

```diff
@@ -4194,6 +4194,11 @@
         "C1607549631-ASIPS"
       ]
     },
+    "VIIRS_SNPP_DayNightBand_At_Sensor_Radiance": {
+      "conceptId": [
```

Comment:
We should probably also update the relevant vis metadata config for this layer with this concept id

Reply:
Yes, probably a few others too. Let me know where to update it.